src/DashboardService.Models.Dto/Responses/GroupResponse.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace LT.DigitalOffice.DashboardService.Models.Dto.Responses;
+
+public record GroupResponse
```

Comment:
you need to put here related entities. Board for example

Reply:
++